controller/src/main/java/io/pravega/controller/task/Stream/StreamTransactionMetadataTasks.java

```diff
@@ -114,7 +128,7 @@ public Void initializeStreamWriters(final ClientFactory clientFactory,
 
     @VisibleForTesting
     public Void initializeStreamWriters(final String commitStreamName, final EventStreamWriter<CommitEvent> commitWriter,
-                                 final String abortStreamName, final EventStreamWriter<AbortEvent> abortWriter) {
+                                        final String abortStreamName, final EventStreamWriter<AbortEvent> abortWriter) {
```

Comment:
There is nothing asynchronous or long running happening here. This might as well be done in the constructor.